addons/expose-registry/expose-registry.addon

```diff
@@ -0,0 +1,18 @@
+# Name: registry
+# Description: Exposes the OpenShift internal registry as route
+# URL: https://docs.openshift.org/latest/install_config/registry/securing_and_exposing_registry.html
+
+oc expose service docker-registry -n default
+openshift admin ca create-server-cert --signer-cert=/var/lib/origin/openshift.local.config/master/ca.crt --signer-key=/var/lib/origin/openshift.local.config/master/ca.key --signer-serial=/var/lib/origin/openshift.local.config/master/ca.serial.txt --hostnames='docker-registry-default.#{routing-suffix},docker-registry.default.svc.cluster.local,172.30.1.1' --cert=/etc/secrets/registry.crt --key=/etc/secrets/registry.key
+openshift cli secrets new registry-secret /etc/secrets/registry.crt /etc/secrets/registry.key
```

Comment:
What openshift cli or admin suppose to execute? I was thinking that for add on we just parse the file and execute commands line by line?